pinecone/core/client/model/index_meta_database.py

```diff
@@ -164,7 +164,7 @@ def _from_openapi_data(cls, *args, **kwargs):  # noqa: E501
                                 through its discriminator because we passed in
                                 _visited_composed_classes = (Animal,)
             name (str): [optional]  # noqa: E501
-            dimensions (str): [optional]  # noqa: E501
+            dimension (str): [optional]  # noqa: E501
```

Comment:
why is it (str)?

Reply:
yeah needs to change in the spec, we don't use it so maybe skip it for this release?